main.py

```diff
@@ -46,8 +46,9 @@ def parse_args():
     )
     parser.add_argument(
         "--modeltype",
-        default="causal",
-        help="AutoModel to use, it can be causal or seq2seq",
+        default="wx",
```

Comment:
Perhaps we could leave this default setting as `causal`, so we don't change the baseline scenario where HuggingFace models are used.

Reply:
I'm not sure about this to be honest, since we'll be using it mainly for wx, and it allows different values for generation parameters (for example, it doesn't support `top_k=0`, which is default for hf)